src/main/java/com/backbase/ct/bbfuel/healthcheck/HealthCheck.java

```diff
@@ -13,7 +13,7 @@
 public class HealthCheck {
 
     private static final String ACTUATOR = "/actuator";
-    private static final String INFO = "/info";
+    private static final String HEALTH = "/health";
```

Comment:
there is no added value for having this split into 2 separate constants (it is private so you can always refactor later when necessary)

Reply:
Done.